### PR TITLE
ALL-1456: add rates submodule

### PR DESCRIPTION
**Pull request by MartinTTM**
# Description
Added `rates` submodule including `getCurrentRate` and `getCurrentRateBatch` methods for obtaining current fiat/crypto exchange rates.

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## [1.4.22] - 2023.06.01
```
Comment (samuelsramko) [inline CHANGELOG.md:1]: 1.4.23

Comment (samuelsramko) [reply CHANGELOG.md:1]: also missing in package.json version upgrade

File: src/service/rate/rates.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    return ErrorUtils.tryFail(async () => {
+      return this.connector.post<Rate>({
+        path: `${this.getUrlPrefix()}`,
+        body: pairs
+      });
+    });
+  }
+
+  private getUrlPrefix() {
+    return this.config.version === ApiVersion.V1 ? 'tatum/rate' : 'rate'
```
Comment (samuelsramko) [inline src/service/rate/rates.ts:46]: we should support only new ApiVersion.V2 here - not fallback to V1 pls

**Review by samuelsramko — CHANGES_REQUESTED** [review]

**Comment by MartinTTM** [comment]
wait for API update to be merged before mergng this

File: src/e2e/tatum.rates.spec.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+      network: Network.ETHEREUM_SEPOLIA,
+      verbose: true,
+      retryDelay: 1000,
+      retryCount: 2,
+      version: ApiVersion.V2,
+    })
+  })
+  it("get ETH/EUR", async () => {
+    const res = await tatum.rates.getCurrentRate("BTC","EUR");
+    console.log(res)
```
Comment (Hathoriel) [inline src/e2e/tatum.rates.spec.ts:17]: Please remove console logs, when your tests are stable, we need to clear the logs.

File: src/service/rate/rates.ts (hunk 2 above)
Comment (Hathoriel) [reply src/service/rate/rates.ts]: I think we should directly go to the v2, because there is no v1.

**Review by Hathoriel — APPROVED** [review]

File: src/e2e/tatum.rates.spec.ts (hunk 3 above)
Comment (MartinTTM) [reply src/e2e/tatum.rates.spec.ts]: removed

**Review by samuelsramko — APPROVED** [review]

**Pull request merged**